docs/guides/upgrading-cluster.md

```diff
@@ -16,16 +16,18 @@ You can update or upgrade your cluster using Terraform.
 
 ## Upgrading your cluster
 
-To upgrade your ROSA cluster to another version, export the following variable then run `terraform apply`.
+To upgrade your ROSA cluster to another version, upgrade your account roles and policies, export the following variables, and then run `terraform apply`.
+
+1. If you are upgrading to a new y-stream, such as upgrading from version 4.12.19 to 4.13.1, run `terraform apply` in the [`account_roles` module](../../examples/create_account_roles/README.md). You must specify your desired y-stream version by using the following variables: 
 
 1. Export the `TF_VAR_openshift_version` with the intended version. Your value must be prepended with `openshift-v`.
-    ```
-    export TF_VAR_openshift_version=<version_number>
-    ```
-1. Upgrading to a new cluster version, especially when transitioning between major y-streams, requires approval. You might be requested to provide administrative confirmation regarding significant modifications for your cluster. In this case, when you first attempt the upgrade, you will receive an error message that provides guidance on the necessary modifications. It is essential to follow those instructions carefully and indicate completion of the requirements by adding the "upgrade_acknowledgements_for" attribute to your resource, specifying the target version. For example, if you are upgrading from 4.11.43 to 4.12.21, you should use '4.12' as the value for this variable.
-    ```
-    upgrade_acknowledgements_for = <version_acknowledgement>
-    ```
+        ```
+        export TF_VAR_openshift_version=<version_number>
+        ```
+1. Upgrading your cluster requires approval, especially when transitioning between major y-streams. You may be required to provide administrative confirmation regarding significant modifications to your cluster. In this case, when you first attempt the upgrade, you will receive an error message that provides guidance regarding the necessary modifications. It is essential to follow the instructions carefully. You indicate completion of the requirements by adding the `upgrade_acknowledgements_for` variable to your Terraform plan with your targeted version. For example, if you are upgrading from version 4.11.43 to 4.12.21, you should use '4.12' as the value for this variable.
```

Comment:
I don't think the "You" in front of "indicate" is necessary. I would just start the sentence with "indicate" to make it more imperative.